python_modules/dagster/dagster/core/types/config_schema.py

```diff
@@ -0,0 +1,38 @@
+from dagster import check
+
+from .config import resolve_to_config_type
+
+
+class InputSchema:
```

Comment:
now may not be the time but i think we should be using abc for this https://docs.python.org/3/library/abc.html